### ISSUE TITLE
Login/Signup page makeover

### ISSUE THREAD

**Issue by krishp007** [bhavy3103/BMark-ECommerce#42]
We can enhance the UI of login and signup pages. 

**My idea is something like this:**
![image](https://github.com/user-attachments/assets/54e24bba-eb84-406d-801a-913f001c955f)

If you have a better design in mind, please let me know

**Comment by bhavy3103** [comment]
yes, you can, but if you want to learn more in-depth and want to work on some good issues in which logic building was done + your coding skill is needed + your knowledge will also grow then let me just know (for advanced issues).
@krishp007 now you can work on it i don't have any issue with that.

**Comment by krishp007** [comment]
Definitely, I would like to work on some advanced issues to improve my logic building. Please let me know of some issues.
